templates/etherpad/0/rancher-compose.yml

```diff
@@ -0,0 +1,37 @@
+.catalog:
+  name: "Etherpad"
+  version: 'v1'
+  description: "Collaboration in really real-time"
+  uuid: etherpad-1
+  questions:
+    - variable: etherpad_title
+      description: "name of the etherpad service"
+      label: "Etherpad title"
+      required: true
+      type: "string"
+    - variable: etherpad_favicon
+      description: "favicon of the etherpad service"
+      label: "Etherpad favicon"
+      type: "string"
+      default: "http://etherpad.org/favicon.ico"
+    - variable: etherpad_admin_password
+      description: "admin password of the etherpad service"
+      label: "Etherpad admin password"
+      required: true
+      type: "password"
+    - variable: public_port
+      description: "public port to access the etherpad service"
+      label: "Public port"
+      required: true
+      default: "80"
+      type: "int"
+    - variable: etherpad_db_password
+      description: "etherpad password of the etherpad database"
+      label: "Database password"
+      required: true
+      type: "password"
+    - variable: etherpad_db_root_password
+      description: "root password of the etherpad database"
+      label: "Database root password"
+      required: true
+      type: "password"
```

Comment:
# Add service combo to select database service. You need to deploy it before.
    - variable: "etherpad_db_service"
      description: |
        Choose the database service to use. 
        It's really recommended a dedicated database service just for Etherpad.
      label: "Database stack/service"
      default: "galera/galera-lb"
      required: true
      type: "service"